app/src/main/res/layout/fragment_home.xml

```diff
@@ -19,141 +20,152 @@
-
-                <TextView
-                    app:layout_constraintTop_toBottomOf="@id/contact_alerts_title"
-                    app:layout_constraintStart_toEndOf="@id/contact_alerts_icon"
-                    android:id="@+id/contact_alerts_description"
-                    android:layout_width="208dp"
                     android:layout_height="wrap_content"
-                    android:layout_marginStart="16dp"
-                    android:layout_marginTop="12dp"
-                    android:text="@string/home_contact_alerts_card_description"
-                    android:textAppearance="@style/card_view_body"/>
-
+                    android:layout_marginTop="@dimen/default_line_height"
+                    android:layout_marginStart="19dp"
+                    android:text="@string/home_contact_alerts_card_title"/>
                 <TextView
-                    app:layout_constraintTop_toBottomOf="@id/contact_alerts_description"
-                    app:layout_constraintStart_toEndOf="@id/contact_alerts_icon"
-                    android:id="@+id/see_alerts"
-                    android:layout_width="140dp"
+                    android:id="@+id/contact_alerts_description"
+                    style="@style/card_view_body"
+                    android:layout_width="242dp"
                     android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/default_line_height"
                     android:layout_marginStart="19dp"
-                    android:layout_marginTop="14dp"
-                    android:text="@string/home_contact_alerts_card_see_alerts_button"
-                    android:onClick="@{() -> vm.onSeeAlertsClick()}"
-                    android:textColor="@color/colorAccent"
-                    android:textAppearance="@style/card_view_link"
-                    android:textStyle="bold"/>
-            </androidx.constraintlayout.widget.ConstraintLayout>
+                    android:layout_marginBottom="@dimen/default_line_height"
+                    android:text="@string/home_contact_alerts_card_description"/>
+            </LinearLayout>
         </androidx.cardview.widget.CardView>
         <!-- ************************************************************ -->
         <!-- ************************************************************ -->
 
+            <ImageView
+                android:id="@+id/alert_icon"
+                app:layout_constraintStart_toStartOf="@id/contact_alerts_card"
+                app:layout_constraintTop_toTopOf="@id/contact_alerts_card"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:src="@drawable/ic_alert_circle"
+                android:elevation="2dp"
+                android:translationY="-8dp"
+                android:translationX="-8dp"
+                android:visibility="@{vm.newAlerts ? View.VISIBLE : View.GONE}"/>
```

Comment:
You can use isVisible property to bind directly to a boolean (see `XmlBindingAdapters`).